### DISCUSSION TITLE
Renal and total body clearance in Mobi

### DISCUSSION THREAD

**Discussion by bahsa505** [Open-Systems-Pharmacology/Forum#1266] [Questions & Problems]
Hi, 

I have a couple of questions on defining renal clearance in Mobi.  I am working on a compound that is eliminated both by hepatic metabolism and renal clearance. In Mobi I can see the hepatic clearance parameters at `Reactions`,  I can't find out though where to find the renal clearance parameters? Is there a way also to find the total body clearance?
Another question, if I am to define the total body clearance as a function of creatinine clearance, how this could be implemented in Mobi?

Thanks

**Comment by AndreDlm** [comment]
That depends on the type of renal clearance. If you did not model it explicitly using transporters, you can find the renal clearances in the `passive transports` building block: 
![image](https://user-images.githubusercontent.com/26541603/182188571-c93741cb-a568-4647-ad11-703ea56d2e20.png)

The clearance rate is then defined in the tabs `Kinetic`, e.g. in case of first order tubular secretion here:
![image](https://user-images.githubusercontent.com/26541603/182188986-3d4f1414-24df-42ff-af30-b5f95be69b1e.png)
Note that some parameters in this equation (e.g. `TSspec`) are defined directly in the tab `Parameters`.

**Comment by Scorrt542** [comment]
Hello, have you solved the problem of creatinine clearance?
